docs/start/index.md

```diff
@@ -114,11 +114,11 @@ and explore key features.
 :hidden:
 
 first-steps
-going-further
 modelling/index
 query/index
-Ingesting data <../ingest/index>
+ingest
 application/index
+going-further
```

Comment:
It was intentional that this was below "First Steps".  It should perhaps just be moved into "First Steps" as it's basically an overview of the remaining TOC + some. Having it in the end seems very odd as that's after half the references in there.

Reply:
Something more needs done, but for this iteration I prefer to keep it as it was.

Reply:
It leads to the strange navigation behaviour you had an admonition about. Because there wasn't an index page about "Ingest" in this subsection, we quickly linked to the existing one when ramping up the patch the other day.

Now, to resolve the quirk, the page has been provided based on a summary of the target page as a teaser text. Please change it however you like, possibly on a subsequent iteration, or provide tangible feedback how to improve it.

Reply:
> It was intentional that this was below "First Steps".

You are just referring to the `going-further` item?

Reply:
- GH-368 will resolve it already.